ddp-workspace/projects/ddp-pancan/src/assets/i18n/en.json

```diff
@@ -795,6 +797,7 @@
         "AdminActionPanel": {
             "ActivityEdit": "Click the Edit button to temporarily unlock for editing. Your edits will be saved as you go. When you click submit or navigate away, the activity will be locked again automatically.",
             "EditButton": "Edit"
-        }
+        },
+        "ConfirmDeletion": "Are you sure you want to delete \nthis information?"
```

Comment:
Could you also please make all the changes in es file as well?

Reply:
Thank you for noticing this!
I've added missing Spanish translations.